com.unity.render-pipelines.high-definition/Runtime/Debug/DebugViewMaterialGBuffer.shader

```diff
@@ -119,7 +119,7 @@ Shader "Hidden/HDRP/DebugViewMaterialGBuffer"
                     // TEMP!
                     // For now, the final blit in the backbuffer performs an sRGB write
                     // So in the meantime we apply the inverse transform to linear data to compensate.
-                    if (!needLinearToSRGB)
+                    if (!needLinearToSRGB && _DebugAllowRGBConversion)
```

Comment:
@SebastienLa  Should this be removed now? I had to re-add it when merging master due to compile errors.

Reply:
yes, you can avoid to do change here.
in mainline we have: "if (!needLinearToSRGB && _DebugAOVOutput == 0)"  this change will be backport soon in 10.x.x branch